src/nav/new-relic-solutions.yml

```diff
@@ -50,42 +49,28 @@ pages:
             path: /docs/new-relic-solutions/observability-maturity/innovation-growth/development-quality-implementation-guide
           - title: Release quality
             path: /docs/new-relic-solutions/observability-maturity/innovation-growth/release-quality-implementation-guide
-  - title: New Relic solutions
+  - title: Best practice guides
```

Comment:
Couldn't add a comment above for some reason. 

What if we changed the nav title "Observability maturity" to something more active like "Achieve observability maturity"?

"Discover our best practices"
"Use our cloud migration solutions"

Solve common issues is already there :)

Reply:
i agree. i'll think more about this one later. I'll be reviewing the docs more shortly.